server/controllers/paymentController.js

```diff
@@ -0,0 +1,158 @@
+      message: 'Subscription canceled',
+      subscription: unSubscribe
+    });
+  }
+
+  /**
+   * @static
+   * @description Get a list of users
+   * @param {object} req express request object
+   * @param {object} res express response object
+   * @returns {object} returns object response
+   * @memberof UserController
+   */
+  static async getSubscription(req, res) {
+    const subscriptions = await getAllSubscriptions();
+    return successStat(res, 200, 'subscriptions', subscriptions);
+  }
+
+  /**
+   * @static
+   * @description Get a list of users
+   * @param {object} req express request object
+   * @param {object} res express response object
+   * @returns {object} returns object response
+   * @memberof UserController
+   */
+  static async createPlan(req, res) {
+    const { name } = req.body.plan;
+    const plan = await createPlan(req.body.plan);
+    return successStat(res, 201, 'result', {
+      message: `${name} plan Successful created`,
+      plan
+    });
+  }
+
+  /**
+   * @static
+   * @description Get a list of users
+   * @param {object} req express request object
+   * @param {object} res express response object
+   * @returns {object} returns object response
+   * @memberof UserController
+   */
+  static async deletePlan(req, res) {
+    const planId = req.params.id;
+    const deletedPlan = await deletePlan(planId);
+    return successStat(res, 200, 'result', {
+      message: 'Subscription canceled',
+      plan: deletedPlan
+    });
+  }
+
+  /**
+   * @static
+   * @description Get a list of users
+   * @param {object} req express request object
+   * @param {object} res express response object
+   * @returns {object} returns object response
+   * @memberof UserController
+   */
+  static async getPlan(req, res) {
+    // const subscription = await Subscriptions.findOne({ where: { userId: id } });
+    const plans = await getAllPlans();
+    return successStat(res, 200, 'subscriptions', plans);
+  }
+
+  /**
+   * @static
+   * @description Get a list of users
+   * @param {object} req express request object
+   * @param {object} res express response object
+   * @returns {object} returns object response
+   * @memberof UserController
+   */
+  static async deleteCustomer(req, res) {
+    const { userId } = req.body.user;
+    const customer = await Customer.findOne({ where: { userId } });
+    const { customerId } = customer.dataValues;
+    const deletedCustomer = await deleteCustomer(customerId);
+    return successStat(res, 200, 'result', {
+      message: 'Subscription canceled',
+      customer: deletedCustomer
+    });
+  }
+
+  /**
+   * @static
+   * @description Get a list of users
+   * @param {object} req express request object
+   * @param {object} res express response object
+   * @returns {object} returns object response
+   * @memberof UserController
+   */
+  static async createToken(req, res) {
+    const token = await createToken(req.body.card);
```

Comment:
Really nice job guys. I'd love to test this line locally, what exactly will I be providing from my card? Is it just the `card number` or will I also be adding the `validity date` and `CVV`?

Reply:
The card number for testing is 4242424242424242. I will add it to the description.

Reply:
Nice one bro. You guys did a really good job with the subscription.